### PR TITLE
Update ArtStation

### PR DESCRIPTION
**Pull request by ApCoder123**
Updated img to svg in process

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -12,10 +12,11 @@ websites:
...
   - name: ArtStation
     url: https://www.artstation.com/
-    twitter: ArtStationHQ
-    facebook: artstationhq
-    email_address: support@artstation.com
-    img: artstation.png
+    img: artstation.svg
+    tfa:
+      - email
+    doc: https://help.artstation.com/en/articles/220-setting-up-two-factor-authentication
```
Comment (phallobst) [inline _data/social.yml]:
```suggestion
    doc: https://help.artstation.com/en/articles/220
```

**Review by phallobst — CHANGES_REQUESTED** [review]
The icon is overflowing the viewbox; please replace it with this one:
img

**Comment by ApCoder123** [comment]
Ok, have changed the link and image

**Comment by phallobst** [comment]
On the doc page it says:
> To complete the set up, you will need to confirm via an email that will be sent to you. Please check your spam folder as it may end up in there. Once enabled, you can confirm or deny any log-in requests from the Two Factor Auth Requests page located at the top of your mobile menu bar.

This seems to indicate that only the _set-up_ is done with an email confirmation, but any subsequent 2FA is performed via the app. Can you confirm this?

**Comment by ApCoder123** [comment]
Yes, I can confirm that you are correct. According to [this blog post](https://magazine.artstation.com/2020/07/protect-your-account-with-two-factor-authentication/)

> You can now enable a Two-Factor Authentication (2FA) system using the ArtStation App as an authentication device.

so I have updated tfa to proprietary and added info in the exception about the email confirmation

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**